api_spec/edge_api.json

```diff
@@ -91,6 +94,40 @@
                     }
                 }
             }
+        },
+        "/bulk-identities": {
+            "post": {
+                "tags": [],
+                "summary": "Bulk insert identities",
+                "description": "Bulk uploading identities and traits",
+                "requestBody": {
+                    "content": {
+                        "application/json": {
+                            "schema": {
+                                "type": "object",
+                                "example": {
+                                    "data": [
+                                        {
+                                            "$ref": "#/components/schemas/Sample"
+                                        },
+                                        {
+                                            "$ref": "#/components/schemas/Sample"
+                                        }
+                                    ]
+                                },
+                                "items": {
+                                    "$ref": "#/components/schemas/BulkIdentitiesData"
+                                }
+                            }
+                        }
+                    }
+                },
+                "responses": {
```

Comment:
There are actually a number of other possible responses (all error responses) that could occur from a request to this endpoint. 

One example is here:
https://github.com/Flagsmith/edge-api/blob/main/src/handlers.py#L95

Let's go through this later to determine the others.